aztk/spark/cli/cluster_ssh.py

```diff
@@ -51,16 +54,24 @@ def execute(args: typing.NamedTuple):
     log.info("-------------------------------------------")
 
     # get ssh command
-    ssh_cmd = aztk.cluster.ssh_in_master(
-        cluster_id=ssh_conf.cluster_id,
-        webui=ssh_conf.web_ui_port,
-        jobui=ssh_conf.job_ui_port,
-        jupyter=ssh_conf.jupyter_port,
-        username=ssh_conf.username,
-        host=ssh_conf.host,
-        connect=ssh_conf.connect)
+    try:
+        ssh_cmd = utils.ssh_in_master(
+            client=aztk.client,
+            cluster_id=ssh_conf.cluster_id,
+            webui=ssh_conf.web_ui_port,
+            jobui=ssh_conf.job_ui_port,
+            jupyter=ssh_conf.jupyter_port,
+            username=ssh_conf.username,
+            host=ssh_conf.host,
+            connect=ssh_conf.connect)
 
-    if not ssh_conf.connect:
-        log.info("")
-        log.info("Use the following command to connect to your spark head node:")
-        log.info("\t%s", ssh_cmd)
+        if not ssh_conf.connect:
+            log.info("")
```

Comment:
minor annoyance on the log.info method... I'd argue that log.info() [note no params] should just 'do the right thing'. It's annoying that it doesn't.

Reply:
yeah, agreed. I'll look into it.